docs/command-reference/server-management/slowlog-reset.md

```diff
@@ -0,0 +1,20 @@
+---
+description:  Discover how to use Redis SLOWLOG RESET command to clear all the entries from the slow log.
+---
+
+import PageTitle from '@site/src/components/PageTitle';
+
+# SLOWLOG RESET
+
+<PageTitle title="Redis SLOWLOG RESET Command (Documentation) | Dragonfly" />
+
+## Syntax
+
+    SLOWLOG RESET
```

Comment:
We need to add the ACL categories.

https://redis.io/commands/slowlog-reset/

Don't do it now. The problem is that for sub commands, we don't yet store them in the command registry. We only store the command (`slowlog`) which then dispatches to the subcommand when invoked. This is problematic for two reasons:

1. ACL's are per command (which assumes that subcommands are actual commands stored in the command registry)
2. The subcommand arguments are validated after we call the `top` level command. So you could for example, run `SLOWLOG nonsense` and it would first go over the command registry, find `SLOWLOG` run the callback which then reads `nonsense` and returns an error that there exist no such subcommand. We can do this earlier since we already have the functionality baked in (we would reject it, once we tried to find the command in the registry and not after which would save us a little bit of time). Furthermore, for subcommands with required arguments,`COMMAND SUBCOMMAND some_required_key`, as is now,  the arity check is done within the subcommand handler. On the other hand, if the subcommand was treated as an actual command, the validation step before the InvokeCmd  would take care of the checks automatically. This is more efficient and easier since it doesn't involve hand written arity/validation checks within the subcommand handlers -- which will clean up some functions.

For this reason, I implemented the `ACL` subcommands as explained above. We already have the functionality necessary to apply this to all subcommands but I haven't had a chance yet to write the refactor.

Reply:
I think that at least for `SLOWLOG` is an overkill. The whole command should be ADMIN, since it's around the same functionality.

Reply:
That was a general commend, I was talking about all subommands in general. Also, subcommands might have different `ACL` categories. We can't encapsulate that now, but as I said it's beyond the scope of this PR -- just thinking out loud